docs/source-app/workflows/access_app_state/access_app_state.rst

```diff
@@ -0,0 +1,61 @@
+.. _access_app_state:
+
+################
+Access App State
+################
+
+**Audience:** Users who want to know how the App State can be accessed.
+
+**Level:** Basic
+
+**********************
+What is the App State?
+**********************
+
+In Lightning, each component is stateful and their state is composed of all attributes defined within their **__init__** method.
+
+The **App State** is the collection of all the components' states forming the App.
+
+************************************
+What is special about the App State?
+************************************
+
+The **App State** is always up-to-date, even running an App in the cloud on multiple machines.
```

Comment:
The spacing between the lines here will make it awkward to read the text, as each sentence will be formatted as its own paragraph.

Reply:
Updated